actionpack/test/controller/redirect_test.rb

```diff
@@ -236,18 +244,52 @@ def test_redirect_to_url_with_network_path_reference
 
   def test_redirect_to_back
     @request.env["HTTP_REFERER"] = "http://www.example.com/coming/from"
-    get :redirect_to_back
+
+    ActiveSupport::Deprecation.silence do
+      get :redirect_to_back
+    end
+
     assert_response :redirect
     assert_equal "http://www.example.com/coming/from", redirect_to_url
   end
 
+  def test_redirect_to_back_is_deprecated
```

Comment:
We don't need an additional test here, the previous tests changed to `assert_deprecated` are enough.